dbt/models/iasworld/schema/iasworld.addn.yml

```diff
@@ -9,6 +9,10 @@ sources:
     tables:
       - name: addn
         description: '{{ doc("table_addn") }}'
+        freshness:
+          filter: taxyr >= date_format(current_date - interval '1' year, '%Y')
```

Comment:
Not all tables have a `taxyr` column, but for those that do I'm filtering the table to only scan the latest two years to limit data scanned.